src/Criteo.OpenApi.Comparator/ComparisonRules.cs

```diff
@@ -115,7 +115,7 @@ public static class ComparisonRules
             Id = 1006,
             Code = nameof(RemovedDefinition),
             Message =
-                "The new version is missing a definition that was found in the old version. Was '{0}' removed or renamed?",
```

Comment:
This means we won't see the missing schema name anymore. So it will be very hard to find :)

Reply:
Harder to find I agree, but very hard to find I am not so sure, as the comparator provides an `OldJsonRef` / `NewJsonRef` for each detected difference

Reply:
I've updated my commit, I've added back the schema name in the error message, it will be empty when the schema is inlined in a response or a request